### PR TITLE
Property check mechanism

### PR DESCRIPTION
**Pull request by anutre**
Needed to check if the requested property exists

### CODE REVIEW

File: src/Http/Response.php [hunk 1]
```diff
@@ -30,16 +30,20 @@ function __construct(ResponseInterface $response)
...
      * @return mixed
      */
     function __get($name)
     {
-        return $this->data->$name;
+        if (property_exists($this->data, $name)) {
+            return $this->data->$name;
+        }
+
+        return $this->data;
```
Comment (atanasiuk-hubspot) [inline src/Http/Response.php]: How can we check in application code if property value or $data is returned? Currently we can:

```php
$response = ...
$username = $response->username;
if (!empty(username)) {
  // property exists and not empty
}

Comment (anutre) [reply src/Http/Response.php]: I expect to receive the necessary data from the SDK and get an error or notification if there are no data.

Comment (atanasiuk-hubspot) [reply src/Http/Response.php]: As I can see provided solution doesn't throw any errors

Comment (anutre) [reply src/Http/Response.php]: I agree, if such a decision is made by the community, I can make changes.

**Pull request closed without merging**